cli/fetch_geocode.py

```diff
@@ -1,8 +1,12 @@
 import sys
 import os
+import django
 
 sys.path.append(os.path.join(os.path.dirname(os.path.abspath(__file__)), ".."))
 
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "citystreets.settings")
+django.setup()
+
```

Comment:
What does this do?

Reply:
Because geocode depends on our django model (CachedSearch), we need to set up the django app to run our script

Reply:
ah gotcha